CourseChamp-Frontend/src/components/ViewReview.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+    <div class="course-rating">
+      <div class="rating-header">
+        <h1>{{ rating }}/5</h1>
+        <p>Overall Quality Based on {{ numberOfRatings }} ratings</p>
+      </div>
+      <div class="course-code">
+        <h2>ECSE 428</h2>
+      </div>
+      <div class="review">
+        <p>{{ reviewText }}</p>
```

Comment:
If this is for the overall course rating, why are you putting the review text here?

Reply:
its a review by one user, so for each review, we will store the text and put it there. All of this is code for one review

Reply:
Ah ok so lines 1-9 is as a header for the course in general and then 10-end is the specific review?